releases/prod/app.yml

```diff
@@ -24,6 +24,7 @@ spec:
     enable: true
   values:
     appealReplyServiceApi:
+      allowCreate: 'false'
```

Comment:
Can this default to `'true'` in https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/master/charts/app/values.yaml#L89 ?

Reply:
It defaults to true elsewhere - this is just for Prod where it should be false